### PR TITLE
add support for assignee search, #629

### LINKED ISSUE #629: Nach Bearbeiter*innen filtern
Die Suche nach Bearbeiter*innen unter Gruppenaufgaben in Bearbeitung funktioniert nicht mehr. Es kann nur nach Aufgaben-Titel und Beschreibung gesucht werden.
Wahrscheinlich funktioniert dies seit dem Umstieg auf das neue Tasklist-Backend schon nicht mehr.

## ToDo

* Es soll ein Bearbeiter Filter eingeführt werden (User-Select)
     * Keine Einschränkung auf Personen 
* Damit die Filter übersichtlich bleiben, soll eine neue Filterfunktion eingebaut werden
     * Filter Button mit Öffnung DropDown
     * Tag und Bearbeiter feste Filter im DropDown
     * Beim Zuklappen sollen Filter-Chips angezeigt werden, die gelöscht werden können 
     * Filter analog zur Suche und Tag in der Session gespeichert (RAM)
* Annahme: Polyflow kann bei GroupTasks nach Bearbeiter filtern 

## Akzeptanzkriterien
- Es kann sowohl wie aktuell nach Titel und Beschreibung, aber auch wieder nach Bearbeiter gesucht werden.

**Additional Context**

* https://git.muenchen.de/digitalisierung/digiwf-support/-/issues/362
* https://adresssuche-test.muenchen.de
 
## Abgrenzung
- Die Komponente für die Auswahl der Benutzer wird so wie die heute in den Formularen verwendet wird, genutzt. Die Versorgung mit Benutzern (aus LDAP) wird über die Engine realisiert (bestehende legacy User SST). In der Zukunft wird diese Schnittstelle separat von der Engine bereitgestellt. Dazu gibt es später eine neue Story.
-

### PR DESCRIPTION
**Pull request by zambrovski**
### Description

- searches task.assignee in assigned tasks.
- provides new query parameter for task list search API

### Reference

Issues: closes #629 

### Check-List

- [x] All Acceptance criteria of user story are met
- [x] JUnit tests are written (60% CodeCov)
- [x] [Internal Review]([https://github.com/it-at-m/digiwf-core/blob/dev/CHANGELOG.md](https://confluence.muenchen.de/display/MPdZ/Review+-+DigiWF)) is maintained
- [ ] Documentations [external](https://github.com/it-at-m/digiwf-core/tree/dev/docs) and [internal](https://wiki.muenchen.de/betriebshandbuch/index.php?title=DigiWF&sfr=betriebshandbuch) are completed
- [ ] Smoketest successful (Manual E2E-Test depending on Change) 
- [ ] No Branch waste left
- [ ] [Board](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) is up-to-date
- [ ] Openshift environments are prepared (Secrets, etc.) and release-issue is maintained

### CODE REVIEW

File: digiwf-apps/packages/apps/digiwf-tasklist/src/components/common/UserSearchInput.vue [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<template>
+
+</template>
+
+<script setup lang="ts">
+
+</script>
+
+
```
Comment (darenegade) [inline digiwf-apps/packages/apps/digiwf-tasklist/src/components/common/UserSearchInput.vue]: Ist das eine Leiche😅

File: digiwf-task/pom.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
     <properties>
         <kotlin.version>1.9.22</kotlin.version>
         <camunda.version>7.20.0</camunda.version>
-        <polyflow.version>4.1.0</polyflow.version>
+        <polyflow.version>4.1.1-SNAPSHOT</polyflow.version>
```
Comment (darenegade) [inline digiwf-task/pom.xml]: Snapshot noch entfernen

File: digiwf-apps/packages/apps/digiwf-tasklist/src/components/common/UserSearchInput.vue (hunk 1 above)
Comment (StephanStrehlerCGI) [reply digiwf-apps/packages/apps/digiwf-tasklist/src/components/common/UserSearchInput.vue]: entfernt

File: digiwf-task/pom.xml (hunk 2 above)
Comment (StephanStrehlerCGI) [reply digiwf-task/pom.xml]: done

**Review by lehju — APPROVED** [review]
Funktioniert alles 👍🏼

**Pull request merged**